planner_cspace/include/planner_cspace/grid_astar.h

```diff
@@ -68,14 +68,14 @@ class GridAstar
     Vec v_;
 
     PriorityVec()
+      : p_(0)
```

Comment:
Is there no need to initialize p_raw_?

Reply:
This constructor is not used at now. It looks better to be removed.